src/css/normalize+resets.css

```
@@ -193,14 +193,14 @@ button::-moz-focus-inner,
 button:-moz-focusring,
 [type=button]:-moz-focusring,
 [type=reset]:-moz-focusring,
-[type=submit]:-moz-focusring {
+[type=submit]:-moz-focusring  {
   outline: 1px dotted ButtonText;
 }
 
 /**
  * Correct the padding in Firefox.
  */
-fieldset {
+fieldset  {
```

```suggestion
fieldset {
```

Check if there are more double whitespace in the project and remove them where they are not necessary.

---

Checked for double spaces, removed